src/settings.c

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <stdio.h>
```

Comment:
Fixes `warning: implicit declaration of function 'snprintf' [-Wimplicit-function-declaration]`